### ISSUE TITLE
Midpoint Review - Accessibility Feedback - MHV medications list filter

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#94748]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- @shiragoodman pinged me today because my feedback ticket was missing from your Midpoint Review on 9/19. I have no idea what happened! I remember writing up my feedback, but somewhere between me and GitHub something must have gone wrong. I apologize for the very late feedback, and I hope my delay hasn't slowed down your work on this.
- Looking at Figma, it looks like you've already made some changes and (I think) aligned to Ryan Thurlwell's recommended filter/sort pattern. My feedback below is hopefully relevant to where you are now.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** Make sure you're thinking about focus management for when filters are applied. After someone pushes the Filter button, where does keyboard focus go? You want to send it to a place that's most immediately actionable for users, setting them up well for their next step.
   
   Likewise, make sure you're thinking about what kind of feedback is announced to screen reader users when the filter action occurs. This is often solved in tandem with focus management, because wherever you send keyboard focus is immediately announced to screen reader users --- meaning, good focus management can also be good screen reader messaging.
   
   What I usually recommend for something like this is that after pressing the "Filter" button focus moves to the "Showing 1 to X of..." message, which implicitly tells screen reader users that the action has been successful; explicitly tells them what the new state of the page is; and sets up keyboard users so that their next tab stop is in the filtered results.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
